Homework_8_classes_objects_fields_methods/MyAmazingRPG/src/Character.java

```diff
@@ -0,0 +1,51 @@
+public class Character {
+
+    static int characterCount;
+
+    public Character() {
+        characterCount++;
+    }
+
+    String name;
+    int health;
+    int strength;
+    int defense;
+    static int potion = 20;
+    int strengthUp;
+    int criticalHit;
+
+    public void healthUp() {
+        this.health = this.health + potion;
+    }
+
+    public void setStrengthUp() {
+        this.strength = this.strengthUp;
```

Comment:
Why is this needed? I mean `strenghtUp` has no default value - did you mean to increase strength a bit?

This can be done without using additional variable (which is not modified anywhere among the methods)

`this.strengh += 20`